plugins/powershell/help.md

```diff
@@ -153,7 +153,7 @@ Example output:
 
 ## Troubleshooting
 
-Round Robin is not supported
+Currently we do not support the use of round-robin DNS lookups.
```

Comment:
Suggestion - avoid use of 'we':
Note that round-robin DNS lookups are not supported.